test/integration/emitter_test.cpp

```diff
@@ -138,6 +138,70 @@ TEST_F(EmitterTest, EmptyFlowSeq) {
   ExpectEmit("[]");
 }
 
+TEST_F(EmitterTest, EmptyBlockSeqWithBegunContent) {
+  out << BeginSeq;
+  out << BeginSeq << Comment("comment") << EndSeq;
+  out << BeginSeq << Newline << EndSeq;
+  out << BeginSeq << Anchor("test") << EndSeq;
```

Comment:
I think the behavior here should either be:

1. An anchor to a null entry in this sequence, e.g. `[&test ~]`
2. An error.

Similarly for the tag one. What do you think?

Reply:
I prefer the second one. In essence，these two (anchor and tag) are the wrong formats.  yaml-cpp should not emit these.

Reply:
@jbeder I have updated the code. When the empty anchor/tag is emitted, yaml-cpp will set the error message `INVALID_ANCHOR`/`INVALID_TAG`.